extensions-core/druid-lookups/README.md

```diff
@@ -0,0 +1,122 @@
+# Cached Lookup Module
```

Comment:
can u move this doc to https://github.com/druid-io/druid/tree/master/docs/content/development/extensions-core and update https://github.com/druid-io/druid/blob/master/docs/content/development/extensions.md